shell/packages/sandstorm-accounts-packages/accounts.js

```diff
@@ -0,0 +1,50 @@
+if (Meteor.isClient) {
+  Meteor.loginWithGoogle = function(options, callback) {
```

Comment:
I notice that the place that calls `loginWith*` is actually in `accounts-ui`, which we've forked, so we can change this. Considering that the Google and Github versions of this method are identical except for the package they call into, maybe it makes sense to have just one method that either takes the package as a parameter or takes a name and looks up the package in a map?